packages/babel-helpers/src/helpers.js

```diff
@@ -374,6 +374,26 @@ helpers.extends = defineHelper(`
   }
 `);
 
+helpers.spreadByDefine = defineHelper(`
```

Comment:
Can we rename this to `objectSpread`?

Reply:
Done! Also added the same test specified in https://github.com/babel/babel/issues/7382